sentence_transformers/SentenceTransformer.py

```diff
@@ -38,17 +39,27 @@ class SentenceTransformer(nn.Sequential):
     :param modules: This parameter can be used to create custom SentenceTransformer models from scratch.
     :param device: Device (like 'cuda' / 'cpu') that should be used for computation. If None, checks if a GPU can be used.
     :param cache_folder: Path to store models. Can be also set by SENTENCE_TRANSFORMERS_HOME enviroment variable.
-    :param use_auth_token: HuggingFace authentication token to download private models.
+    :param token: HuggingFace authentication token to download private models.
```

Comment:
seems `use_auth_token` is still in the constructor, no need to delete the docstring.

Reply:
I've deleted the docstring because `use_auth_token` will be softly deprecated. I'd rather not have deprecated arguments in the docstring.